arednHow-toGuides/vm-install.rst

```diff
@@ -9,19 +9,19 @@ The use of virtual machines as AREDN® nodes is for advanced users. Most users s
 Prerequisites / Image information
 ---------------------------------
 
-At a minimum the VM must have two virtual CPUs, 64mb memory, and approximately 200mb free storage. Providing more CPU is generally not needed on modern hardware. Extra memory can be useful for a Supernode or large tunnel server, however more than 1gb is not needed.
+At a minimum the VM must have two virtual CPUs, 64mb memory, and 128mb of storage. Providing more CPU is generally not needed on modern hardware.
 
-There are two modes for networking: single-port and multi-port. This is automatically selected based on the number of available network interfaces detected. Set the number of interfaces *before* powering on the VM for the first time.
+There are two modes for networking: single-port and multi-port. Set the number of interfaces *before* powering on the VM for the first time. Regardless of the number of interfaced created, the node will initially be configured in 'single-port mode'. This can be changed later in the AREDN UI.
 
 Single-port mode
   All traffic utilizes VLANs as described in the *Advanced Options* section of the **Node Admin** documentation. This requires your virtual interface to be VLAN aware or to be set as a passthrough interface.
```

Comment:
I couldn't find this reference, but maybe we could link to it?

Reply:
That was fixed a week or so ago, so you may need to pull the latest.

Reply:
It should say the Advanced Options section of the Network Settings dialog in the Node Admin documentation.

Reply:
Ta. I updated the PR.